.eslintrc.js

```diff
@@ -96,6 +96,7 @@ module.exports = {
     // ESLint plugin for prettier formatting
     // https://github.com/prettier/eslint-plugin-prettier
     'prettier/prettier': 'error',
+    'react/destructuring-assignment': 'never',
```

Comment:
`'always'` is preferable value to my -_-

Reply:
I'd also prefer always, makes for cleaner code.

Reply:
Of course, only reason why I used this one was because I didn't want to start changing hundreds lines of code at that time.

Reply:
They're warnings though? If we agree on the rule we can just fix them as we go.

Reply:
I think it didn't let commit otherwise. Anyway if you want to change coding style rules you can create a separate pr of it, this one does not introduce any changes to the current setup.

Reply:
Okay I can do that in next refactor.